### PR TITLE
Bump Whitenoise

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -83,11 +83,11 @@ repos:
...
-          - "eslint-config-prettier@8.3.0"
-  #       Additional excludes in /.eslintignore file
+#  - repo: https://github.com/eslint/eslint
+#    rev: v7.32.0
+#    hooks:
+#      - id: eslint
+#        additional_dependencies:
+#          - "eslint-plugin-json@2.1.1"
+#          - "eslint-config-prettier@8.3.0"
+#  #       Additional excludes in /.eslintignore file
```
Comment (robhudson) [inline .pre-commit-config.yaml:93]: Oops, I didn't mean to commit this.

File: bedrock/settings/base.py [hunk 2]
```diff
@@ -537,6 +537,7 @@ def get_app_name(hostname):
 MIDDLEWARE = [
     "allow_cidr.middleware.AllowCIDRMiddleware",
     "django.middleware.security.SecurityMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
```
Comment (pmac) [inline bedrock/settings/base.py:540]: Why do we need this?

Comment (robhudson) [reply bedrock/settings/base.py:540]: This is the way

Comment (robhudson) [reply bedrock/settings/base.py:540]: The new way that whitenoise is configured. It is removed from the wsgi.py file and added here. See https://whitenoise.evans.io/en/stable/django.html#enable-whitenoise

**Review by pmac — APPROVED** [review]
👍

**Pull request merged**